Sustainsys.Saml2.AspNetCore2/HttpRequestExtensions.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Sustainsys.Saml2.AspNetCore2
 {
-    static class HttpRequestExtensions
```

Comment:
changed to public to allow using ToHttpRequestData extension method on client app